inspector.el

```diff
@@ -806,10 +806,17 @@ When PRESERVE-HISTORY is T, inspector history is not cleared."
     (with-current-buffer-window "*inspector pprint*"
         nil nil
       (local-set-key "q" #'kill-this-buffer)
+      ;; For evil mode use `q to kill the buffer
+        (when (fboundp 'evil-local-set-key)
```

Comment:
The keybinding in this package depends on the function of treeview.el. That mean most of key binding will work except `q`

Reply:
Ok. It is a pity that it is the only evil thing that got in.
How does evil work when you use other packages? You need a minor mode, I guess.

Reply:
Why can't you access the normal "q" key binding from evil-mode?

Reply:
This problem happened with many packages, not just this one. It is hard to know the deep root cause of this problem because I don't have enough skill and experience with the evil source code.

I think that because evil-mode acts like a major mode (in normal Emacs logic, we know that two major modes cannot exist at the same time, but that is how evil works), that means when using it, it will overwrite most of the bindings of the normal Emacs keymap if the user or package creator does not manually add binding using their built-in function like `evil-define-key`.

Reply:
I understand. But it feels wrong modify packages because of some outsider package problem (evil-mode in this case). I'd prefer not to do this.

Reply:
The buffer can be killed using normal `kill-buffer` command binding in any case.

Reply:
Thank you 🥰🌹